backend/app/main.py

```diff
@@ -74,15 +74,10 @@
 # Add CORS middleware to allow cross origin requests from any port
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=[
-        "http://localhost:3000",
-        r"https:\/\/.*\.climatepolicyradar\.org",
-        r"https:\/\/.*\.dev\.climatepolicyradar\.org",
-    ],
     allow_credentials=True,
     allow_methods=["*"],
     allow_headers=["*"],
-    allow_origin_regex=True,
+    allow_origin_regex=r"https:\/\/.*\.climatepolicyradar\.org|http:\/\/localhost:3000",
```

Comment:
worth adding climate-laws.org now?

Reply:
I'll make a note - and slip it in with another PR ;)